stories/SideNav.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import SideNav from '../src/SideNav';
+import SideNavItem from '../src/SideNavItem';
+import Button from '../src/Button';
+
+const stories = storiesOf('Javascript|SideNav', module);
+const reactMaterializeLogo = require('./react-materialize-logo.svg');
+
+stories.addParameters({
+  info: {
+    text: `This is a slide out menu.`
+  }
+});
+
+stories.add('Default', () => (
+  <div>
+    <style>
+      {`
+        #root > div > div {
+          z-index: 99999 !important;
+        }
+      `}
+    </style>
+    <SideNav
+      trigger={<Button>SIDE NAV DEMO</Button>}
+      options={{ closeOnClick: true }}
+      >
+      <SideNavItem userView
+        user={{
+          background: 'https://onluiz.github.io/react-materialize-docs/img/office.jpg',
```

Comment:
Used an image from a different site. Is it ok? I did not want to upload a new image to the project without any discussion

Reply:
We use placeimg or someother service I think

Reply:
Done!